src/js/entry.js

```diff
@@ -19,5 +19,7 @@ document.addEventListener('DOMContentLoaded', () => {
       return require('./nightly').load();
     case 'releases':
       return require('./releases').load();
+    case 'testimonials':
```

Comment:
Do we need to add users.html as well?

Reply:
nope the javascript is for the carousel only.